MetaCell/salk-interactive-atlas
# Feature/213

Depends on https://github.com/MetaCell/geppetto-meta/issues/397

Closes #213 

Implemented solution: 
- Adds example pdf to frontend assets
- Renames electrophysiologyViewer to detailsViewer
- Adds detailsViewer component
- Updates ThreeDViewer to use threeDObjects prop of geppetto canvas instead of manually adding the cells to the scene (so that we can take advantage of the onSelection callback - It will only work after https://github.com/MetaCell/geppetto-meta/issues/397 gets merged )
- Updates unselected tab color

How to test this PR: 

Click on a cell, check if details viewer pops up with the correct information:

https://user-images.githubusercontent.com/19196034/195389953-e5b94fae-edb1-4ca9-a53c-6ce685c465bd.mp4

### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
